src/Products/Annotation/Annotator/TextFieldAnnotator.cs

```diff
@@ -1,58 +1,61 @@
-﻿using GroupDocs.Annotation.Domain;
+﻿using GroupDocs.Annotation.Models.AnnotationModels;
 using GroupDocs.Annotation.MVC.Products.Annotation.Entity.Web;
+using GroupDocs.Annotation.Options;
 using System;
 
 namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
 {
-    public class TextFieldAnnotator : AbstractTextAnnotator
+    public class TextFieldAnnotator : BaseAnnotator
     {
+        private TextFieldAnnotation textFieldAnnotation;
+
         public TextFieldAnnotator(AnnotationDataEntity annotationData, PageData pageData)
             : base(annotationData, pageData)
         {
+            textFieldAnnotation = new TextFieldAnnotation() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372470)